test/extended/image_ecosystem/jenkins_plugin.go

```diff
@@ -153,8 +159,17 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 
 		g.By("kick off the build for the jenkins ephemeral and application templates")
 
+		var licensePrefix, pluginName string
 		newAppArgs := []string{exutil.FixturePath("..", "..", "examples", "jenkins", "jenkins-ephemeral-template.json")}
 		newAppArgs, useSnapshotImage := jenkins.SetupSnapshotImage(jenkins.UseLocalPluginSnapshotEnvVarName, localPluginSnapshotImage, localPluginSnapshotImageStream, newAppArgs, oc)
+		if !useSnapshotImage {
+			newAppArgs, useSnapshotImage = jenkins.SetupSnapshotImage(jenkins.UseLocalLoginPluginSnapshotEnvVarName, localLoginPluginSnapshotImage, localLoginPluginSnapshotImageStream, newAppArgs, oc)
+			licensePrefix = loginLicenseText
+			pluginName = loginPluginName
+		} else {
+			licensePrefix = originalLicenseText
+			pluginName = originalPluginName
+		}
```

Comment:
this feels a little weird since there's a precedence being defined (use pipelineplugin snapshot image first, then local login plugin snapshot, then normal image).  At a minimum i'd probably rewrite it as a switch that checks for each type of localimage indicator.

Also probably needs some comments that explain how this test is being overloaded/reused to test the snapshot images from the plugin PRs.